protocols/luci-proto-pppossh/luasrc/model/network/proto_pppossh.lua

```diff
@@ -0,0 +1,48 @@
+-- Copyright 2018 Florian Eckert <fe@dev.tdt.de>
+-- Licensed to the public under the Apache License 2.0.
+
+local netmod = luci.model.network
+
+local proto = netmod:register_protocol("pppossh")
+
+function proto.get_i18n(self)
+	return luci.i18n.translate("PPPoSSH")
+end
+
+function proto.ifname(self)
+	return "pppossh-" .. self.sid
+end
+
+function proto.opkg_package(self)
+	return "pppossh"
+end
+
+function proto.is_installed(self)
+	return nixio.fs.access("/lib/netifd/proto/pppossh.sh")
+end
+
+function proto.is_floating(self)
+	return true
+end
+
+function proto.is_virtual(self)
+	return true
+end
+
+function proto.get_interfaces(self)
+	if self:is_floating() then
+		return nil
+	else
+		return netmod.protocol.get_interfaces(self)
+	end
+end
+
+function proto.contains_interface(self, ifc)
+	if self:is_floating() then
+		return (netmod:ifnameof(ifc) == self:ifname())
+	else
+		return netmod.protocol.contains_interface(self, ifc)
+	end
+end
```

Comment:
ditto

Reply:
The luci-proto-pppossh is floating so we only have to change the functions to

> return (netmod:ifnameof(ifc) == self:ifname())